content/swift/concepts/collections.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: 'Collections'
+Description: 'Collections in Swift are data structures that store multiple values of the same type.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Collections'
+  - 'Arrays'
+  - 'Dictionaries'
+  - 'Sets'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**Collections** are data structures that store multiple values of the same, or different types.
+
+Collections are can be value types or reference types, depending on the implementation.
+
+A value type is a type whose instances are copied when they are assigned to a variable or constant, or when they are passed to a function. A reference type is a type whose instances share a single copy of the data, and use references to access it.
+
+Swift provides three primary collection types: arrays, sets, and dictionaries.
+
+## Syntax
+
+In Swift, collections are defined using different syntaxes, depending on the type of collection.

```

Comment:
I would suggest leaving the descriptions as is, but having one code block before the "append, insert, remove" section with all the syntaxes documented instead of the individual inline code